recipes/traceback-with-variables/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "traceback-with-variables" %}
+{% set version = "1.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/b5/a7/50e61fccfc2c5335d6e0fb71cfc3f41e89a82432f859fdc5da70bc4cb54c/traceback-with-variables-1.1.7.1.tar.gz
```

Comment:
Please use the canonical pypi url found here:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14